src/devices/DimmableLightingDevice/DimmableLightingDevice.ts

```diff
@@ -125,6 +69,7 @@ export default class DimmableLightingDevice extends InsteonDevice {
 					}
 					break;
 				case 0x18: this.emitRemote(['dim','stoppedChanging'], data); break;
+				case 0x21: this.emitRemote(['switch','instant'], data); break;
```

Comment:
Not sure if this should be `'switch','instant'` or `'switch','on'` or `'switch','on','instant'`. I'm not sure if you would ever want to subscribe to just the instant event.

Reply:
'switch','on','instant'

With eventemitter if you sub to `'switch', 'on'` it should subscribe you to everything under it as well

Reply:
Events renamed